_includes/configuration/content-style.md

```diff
@@ -1,12 +1,12 @@
-## content_style
+## `content_style`
 
 This option allows you to set custom CSS styles as a string that is injected into the editor's iframe.
 
 These styles are useful for previewing content as it appears outside the editor. Since these styles are not saved with the content, it is important that they match the styles in the location that the content appears.
 
 **Type:** `String`
 
-##### Example
+### Example: Applying one CCS style using `content_style`
```

Comment:
Is it meant to be `CSS` instead of `CCS`? Same applies to below.

Reply:
:facepalm: Thanks!

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1707/commits/c5fa87a7ac57b7e8aee77308aeee97f343f9054d